plugins/lookup/hashi_vault.py

```diff
@@ -38,13 +38,15 @@
     token_path:
       description: If no token is specified, will try to read the token file from this path.
       env:
-        - name: HOME
+        - name: VAULT_TOKEN_PATH
```

Comment:
This is a breaking change (and would have to wait for 2.0.0), since it stops existing playbooks/roles from working. Just adding HOME probably already is, since HOME is always set and so this will **always** overwrite the INI setting.

It is probably better to keep the current version, and add in the documentation that the plugin will use `$HOME` as a fallback if it isn't specified. (That has to be done manually in code I think.)